google-spreadsheet-config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "spreadsheetId": "1QlEqI-HMumoSDp6HO9tSkLWluoioIZz4U1q1z0DAFD0",
```

Comment:
These IDs also only make sense for our organization... Shouldn't be hardcoded in a public repo like this one. Let us move them to a private repo, yes?